CMakeLists.txt

```diff
@@ -108,6 +108,7 @@ else()
 endif()
 if(CMAKE_BUILD_TYPE MATCHES Debug)
   set(DEBUG 1)
+  add_definitions(-DABORT_ON_INTERNAL_ERROR)
```

Comment:
@neovim/core FYI, setting `ABORT_ON_INTERNAL_ERROR` for "Debug" build-type by default.